argocd/eoepca/mlops/parts/app-mlflow-sharinghub.yaml

```diff
@@ -11,13 +11,13 @@ spec:
   source:
     repoURL: https://github.com/csgroup-oss/mlflow-sharinghub.git
     path: deploy/helm/mlflow-sharinghub
-    targetRevision: main
+    targetRevision: "0.2.0"
     helm:
       valuesObject:
         image:
           repository: eoepca/mlflow-sharinghub
           pullPolicy: Always
-          tag: latest
+          tag: "0.2.0"
```

Comment:
It is better if the version of the image is included (as the default) within the versioned helm chart - so it doesn't need to be specified by the user of the helm chart.

Reply:
Typically the Chart `appVersion` is used to specify the (default) container image.

Reply:
I updated the charts to use "0.2.0" by default.